src/css/ingame_hud/statistics.scss

```diff
@@ -103,6 +104,61 @@
 
                 @include S(padding, 0, 3px);
             }
+
+            button.pin {
+                @include S(width, 12px);
+                @include S(height, 12px);
+                @include S(padding, 0px);
+                background: uiResource("icons/pin.png") center center / 95% no-repeat;
+                position: absolute;
+
+                opacity: 0.6;
+                transition: opacity 0.12s ease-in-out;
+                opacity: 0.6;
+                cursor: pointer;
+                pointer-events: all;
+                @include IncreasedClickArea(5px);
+                transition: opacity 0.12s ease-in-out;
+
+                @include DarkThemeInvert;
+
+                $disabledOpacity: 0.2;
+                $enabledOpacity: 0.6;
+
+                &:hover {
+                    opacity: $enabledOpacity + 0.1;
+                }
+
+                &.alreadyPinned {
+                    opacity: $disabledOpacity !important;
+
+                    &:hover {
+                        opacity: $disabledOpacity + 0.1 !important;
+                    }
+                }
+
+                &.isGoal {
+                    background: uiResource("icons/current_goal_marker.png") center center / 95%
+                        no-repeat;
+                    opacity: $disabledOpacity !important;
+                    cursor: default;
+                    pointer-events: none;
+                }
+
+                &.pinned {
+                    opacity: $disabledOpacity;
+                    &:hover {
+                        opacity: $disabledOpacity + 0.1;
+                    }
+                }
+
+                &.unpinned {
+                    opacity: $enabledOpacity;
+                    &:hover {
+                        opacity: $enabledOpacity + 0.1;
+                    }
+                }
```

Comment:
I had to remove the animation for this screen since it just caused the pin button to continuously pulse. I couldn't figure out how to fix it.

Reply:
Thats fine, I know why